### ISSUE TITLE
Compressed Iron armor missing on body texture FTB Unstable 1.21

### ISSUE THREAD

**Issue by JaspervArkel** [FTBTeam/FTB-Modpack-Issues#5615]
### Modpack

Other...

### Modpack version

1.0.1

### Has the pack been modified

No

### Describe the bug

The Compressed iron armor set  from  pneumaticcraft : Represurrised. don't have any texture for when you are wearing it,
the only thing you are seeing is the purple black missing texture visual.

### Steps to reproduce

1. start wearing the compressed iron armor set
2. look at you player character

### Expected behaviour

you will see that your entire character is covered by purple and black

### Screenshots

![image](https://github.com/user-attachments/assets/3924e028-3e32-4827-874b-e7d0bfda9899)

**Comment by DanielAsper** [comment]
Hello everyone! I wanted to add that I've noticed the issue isn't with the modification itself. I added my own external modification outside of the pack, which includes armor, but the texture of this armor also doesn't display correctly—it just shows up as a big square. So, in my opinion, the problem isn't with the modification, but rather with the internal settings of the modpack.

![Zrzut ekranu 2024-08-24 180839](https://github.com/user-attachments/assets/cafeef3e-b8d9-4e46-9e3d-05856f339a63)

Mod: Mystical Agriculture
 By
 BlakeBr0
